classes/mage.lua

```diff
@@ -13,9 +13,6 @@ local UnitHealth = UnitHealth
 
 local clearcastingVariants; -- Lazy init in lazyCreateClearcastingVariants()
 
-local pyroblast = 11366; -- Pyroblast, the base Pyro spell
-local pyroblastBang = 92315; -- Pyroblast!, a specific spell for instant Pyro introduced in Cataclysm

```

Comment:
These lines must not be removed because code depends on it:
- as glowing button identifier in `registerClass`
- as spell options in `loadOptions`